pkg/backup/item_backupper.go

```diff
@@ -188,7 +189,8 @@ func (ib *itemBackupper) backupItem(logger logrus.FieldLogger, obj runtime.Unstr
 	name = metadata.GetName()
 	namespace = metadata.GetNamespace()
 
-	if groupResource == kuberesource.PersistentVolumes {
+	// #4758 Do not take snapshot for PV to avoid duplicated snapshotting, when CSI feature is enabled.
```

Comment:
This is problematic because the PV may not be provisioned by CSI driver.

You need to explicitly mark the PV in CSI snapshot flow so that you can skip the `take PV action`